pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/PersistentOfflineTopicStats.java

```diff
@@ -23,19 +23,27 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
 
 /**
  * This object is populated using meta data in zookeeper without actually bringing the topic online.
  */
 public class PersistentOfflineTopicStats {
     /** Space used to store the messages for the topic (bytes). */
-    public long storageSize;
+    @Getter
+    @Setter
+    private long storageSize;
 
     /** Total number of messages. */
-    public long totalMessages;
+    @Getter
+    @Setter
+    private long totalMessages;
 
     /** Total backlog. */
-    public long messageBacklog;
+    @Getter
+    @Setter
+    private long messageBacklog;
```

Comment:
These are most likely a breaking change in the public API. In other stats classes there's a separate "Impl" class and a separate interface. I don't see a way how to preserve API compatibility and perhaps we should just break the API when making the change.